### PR TITLE
[Redis] New Param added to choose redis version

### PR DESCRIPTION
**Pull request by gorandom**
## Description

Adding a new Param "RedisVersion" to the New-AzRedisCache command.
Also updating the .NET SDK to the latest version.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RedisCache/RedisCache/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Added `RedisVersion` parameter in `New-AzRedisCache`
```
Comment (MECHANDR) [inline src/RedisCache/RedisCache/ChangeLog.md:21]: `RedisVersion` should also be added to `Set-AzRedisCache`

Comment (gorandom) [reply src/RedisCache/RedisCache/ChangeLog.md]: Added

**Review by isra-fel — CHANGES_REQUESTED** [review]
Please make sure the markdown help files have been regenerated using the commands listed [here](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module) whenever you make a cmdlet syntax change

**Review by isra-fel — APPROVED** [review]

**Review by MECHANDR — APPROVED** [review]

**Pull request merged**